alces-flight-tutorials/metadata.yml

```diff
@@ -0,0 +1,75 @@
+---
+install:
+  el6: |
+    yum install -y -e0 nodejs npm yarn
+  el7: |
+    yum install -y -e0 nodejs npm yarn
```

Comment:
I suspect none of these packages are available in EL7 and you'll need to enable EPEL.  e.g. https://github.com/alces-software/clusterware-sessions/blob/master/xfce/metadata.yml#L14-L21

Looks like there's no `yarn` package in EPEL.  I guess that's why you need `npm`, so `yarn` can be installed?  Is `yarn` required after build or can we use `npm run` or something instead?